main/src/main.ts

```diff
@@ -174,6 +174,26 @@ export function createWindow(
         width: windowState.bounds.width,
     })
 
+    // Workaround for cut/copy/paste/close keybindings not working in devtools window on OSX
+    if (process.platform === "darwin") {
+        currentWindow.webContents.on("devtools-opened", () => {
+            currentWindow.webContents.devToolsWebContents.executeJavaScript(`
```

Comment:
This is really cool, didn't know this was possible! 👍

Reply:
Cool, but it doesn't work for "quit" CMD+Q will not close window if focus is on the devtools. It used to in previous version of electron.